src/nyx/features/image_moments.cpp

```diff
@@ -445,3 +534,35 @@ void ImageMomentsFeature::gpu_process_all_rois (const std::vector<int> & Labels,
 }
 #endif
 
+namespace Nyxus
+{
+    void copy_pixcloud_intensities (std::vector<float> & dst, const pixcloud & src)
+    {
+        dst.reserve (src.size());
+        for (auto pxl : src)
+            dst.push_back (float(pxl.inten));
+    }
+
+    /// @brief Applies to distance-to-contour weighting to intensities of pixel cloud. Saves the result in 'realintens' 
+    void apply_dist2contour_weighting(
+        // input & output
+        std::vector<float>& realintens,

```

Comment:
maybe just say `intcloud` to stay consistent with the header?

Reply:
Fixed in the update, check it out